internal/cfgmgr/cfgmgr.go

```diff
@@ -0,0 +1,163 @@
+package cfgmgr
+
+import (
+	"fmt"
+	"strings"
+	"unicode/utf16"
+
+	"github.com/Microsoft/go-winio/pkg/guid"
+	"github.com/pkg/errors"
+)
+
+//go:generate go run ../../mksyscall_windows.go -output zsyscall_windows.go cfgmgr.go
+
+//sys cmGetDeviceIDListSize(pulLen *uint32, pszFilter *byte, uFlags uint32) (hr error) = cfgmgr32.CM_Get_Device_ID_List_SizeA
+//sys cmGetDeviceIDList(pszFilter *byte, buffer *byte, bufferLen uint32, uFlags uint32) (hr error)= cfgmgr32.CM_Get_Device_ID_ListA
+//sys cmLocateDevNode(pdnDevInst *uint32, pDeviceID string, uFlags uint32) (hr error) = cfgmgr32.CM_Locate_DevNodeW
+//sys cmGetDevNodeProperty(dnDevInst uint32, propertyKey *devPropKey, propertyType *uint32, propertyBuffer *uint16, propertyBufferSize *uint32, uFlags uint32) (hr error) = cfgmgr32.CM_Get_DevNode_PropertyW
+
+type devPropKey struct {
+	fmtid guid.GUID
+	pid   uint32
+}
+
+const (
+	_CM_GETIDLIST_FILTER_BUSRELATIONS = uint32(0x00000020)
+
+	_CM_LOCATE_DEVNODE_NORMAL = uint32(0x00000000)
+
+	_DEVPROP_TYPE_STRING        = uint32(0x00000012)
+	_DEVPROP_TYPEMOD_LIST       = uint32(0x00002000)
+	_DEVPROP_TYPE_STRING_LIST   = uint32(_DEVPROP_TYPE_STRING | _DEVPROP_TYPEMOD_LIST)
+	_DEVPKEY_LOCATIONPATHS_GUID = "a45c254e-df1c-4efd-8020-67d146a850e0"
```

Comment:
This value appears to be the GUID for a category of device properties, rather than being specific to location paths. If that's correct, is there a better name we can use?

Reply:
I addressed this in the above comment